Shubham_Sharma/JWT Assignment/UserAuthorization/UserAuthorization/DbModels/EmployeesContext.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.Extensions.Configuration;
+
+namespace UserAuthorization.DbModels
+{
+    public partial class EmployeesContext : DbContext
+    {
+        public IConfiguration _config;
+        public EmployeesContext(IConfiguration config)
+        {
+            _config = config;
+        }
+
+        public EmployeesContext(DbContextOptions<EmployeesContext> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<EmployeeDetails> EmployeeDetails { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
+                optionsBuilder.UseSqlServer(_config["dbConnection"]);
+            }
```

Comment:
#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings. 
Please Go through the link and fix the mistake

Reply:
Removed the connection string from context file and startup file, and stored the connection string in appsettings.json.